manager/system_handler.py

```diff
@@ -157,10 +157,15 @@ def handle_get(cls, **kwargs):
 
         cves_view = SystemCvesView(list_arguments, {"rh_account_number": connexion.context['user'],
                                                     'inventory_id': inventory_id}, args, args, connexion.request.path)
+
+        opted_out = SystemPlatform.get(SystemPlatform.inventory_id == inventory_id).opt_out
         response = {}
         result = []
         for cve in cves_view:
             record = {}
+            # if system was opted-out dont fetch cves
```

Comment:
Change this to condition before the cycle `if not system.opted_out:` then fetch CVEs for better readablity.